setup.cfg

```diff
@@ -2,12 +2,3 @@
 exclude = .git,.eggs,build,venv,env
```

Comment:
flake8 config could not be moved to pyproject.toml since flake8 doesn't support reading config from the file.
open issue - https://github.com/PyCQA/flake8/issues/234
to use pyproject.toml based config for flake8, we can use this library - https://pypi.org/project/Flake8-pyproject/

Reply:
Should we consider including it in this PR?

Reply:
Should we consider including it in this PR?

Reply:
Lastly are we still using this fine anywhere?

Reply:
yes, we are using this file for flake8 config which is run during github ci/cd

Reply:
Yes, I'm aware that flake8 is holding off on `pyproject.toml` adoption. However, did you try out this package? Does it work? I'd kinda like to have everything in the same file. But of course it's also not worth going through hoops for that. So if it works fine, cool, let's use it for now (until official support) - otherwise let's keep `setup.cfg` for now.